azure-pipelines.yml

```diff
@@ -0,0 +1,44 @@
+jobs:
+- job: 'mac'
+  pool:
+    vmImage: 'macOS 10.13'
+  strategy:
+    matrix:
+      Python27:
+        python.version: '2.7'
+      Python37:
+        python.version: '3.7'
+    maxParallel: 2
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+      architecture: 'x64'
+
+  - script: brew install openssl@1.1
+    displayName: 'Install OpenSSL'
+
+  - script: pip install tox codecov
+    displayName: 'Install tox & codecov'
+
+  - script: git clone https://github.com/google/wycheproof
+    displayName: 'Clone wycheproof'
+
+  - script: |
+      set -e
+      set -x
+      if [[ "$PYTHON_VERSION" == "2.7" ]]; then
+        export TOXENV=py27
+      elif [[ "$PYTHON_VERSION" == "3.7" ]]; then
+        export TOXENV=py37
```

Comment:
Is there a way to set these env vars in the matrix entry?

Reply:
Yes you can, all elements of the matrix are automatically set as environment variables.

Reply:
Excellent! Thanks for the PR.

For our Windows builders we compile our own OpenSSL (whenever a new version is released) against VS2010 and VS2015 (in both x86 and x86-64) and then keep those as build artifacts that are used for each CI job. What's the right way to replicate that behavior in Azure Pipelines? Can we set up a separate pipeline that builds OpenSSL, archive it in some fashion, and then have the primary CI job pull down that artifact before we run the cryptography tests?